packages/third-parties/src/ThirdPartyScriptEmbed.tsx

```diff
@@ -19,15 +19,15 @@ export default function ThirdPartyScriptEmbed({
     <>
       {/* insert script children */}
       {children}
-      {/* insert content */}
-      {content && (
+      {/* insert html */}
+      {html && (
```

Comment:
Maybe using `{html ? <jsx> : null}` to avoid js users (without typing) might accidentally passing `0` down here